src/io/dicom.ts

```diff
@@ -260,10 +258,12 @@ export class DICOMIO {
   async buildImage(seriesFiles: File[]) {
     await this.initialize();
 
-    const result = await readImageDICOMFileSeries(
-      seriesFiles.map((file) => sanitizeFile(file))
-    );
+    const inputImages = seriesFiles.map((file) => sanitizeFile(file));
+    const result = await readImageDicomFileSeries(null, {
+      inputImages,
+      singleSortedSeries: false,
+    });
```

Comment:
@jadh4v Using the @itk-wasm/dicom package here. This call results in an error.  When I copy in a debug build of the .wasm, message is:
`CLI::ArgumentMismatch: --input-images: 1 required INPUT_BINARY_FILE:FILE missing`

Any suggestions?

Reply:
Addressing in a PR here: https://github.com/InsightSoftwareConsortium/itk-wasm/pull/886